stage1/rootfs/usr_from_src/Makefile

```diff
@@ -31,10 +34,25 @@ systemd.done: Makefile systemd.src
 
 # TODO(vc): it may make more sense to have the systemd source be a git submodule?
 systemd.src: Makefile patches/*
+	if [ -z "$(RKT_STAGE1_SYSTEMD_SRC)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_SRC undefined"; exit 1; fi
+	if [ -z "$(RKT_STAGE1_SYSTEMD_VER)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_VER undefined"; exit 1; fi
```

Comment:
We should accept an undefined `RKT_STAGE1_SYSTEMD_VER` to take the default branch: it would be practical when testing with a local systemd repository (e.g. `/home/alban/git/systemd/`).

When `RKT_STAGE1_SYSTEMD_VER` is empty, we should just not use `--branch`, and the `../patches/master/*.patch` patchset should be used.